pombola/south_africa/templates/core/organisation_detail.html

```diff
@@ -39,6 +39,22 @@
     {% include "core/_position_listing.html" with positions=positions %}
   {% endif %}
 
+  <div class="pagination">
+    <span class="step-links">
+      {% if positions.has_previous %}
+        <a href="?page={{ positions.previous_page_number }}">previous</a>
+      {% endif %}
+
+      <span class="current">
```

Comment:
Similarly wrapping the current page info inside a `<span>` tag here makes it look like a button. Taking it out makes it appear as text on the page instead, in which case consider removing the full stop as it could look out of place and lazily adding some padding with `&nbsp;` at either end.